app/src/main/res/layout/settings_options_layout.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<GridLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="0dp"
+    android:columnCount="@integer/settings_options_columns"
+    android:rowCount="@integer/settings_options_rows">
+
+    <com.willowtree.vocable.customviews.SettingsButton
+        android:id="@+id/edit_sayings_button"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_rowWeight="1"
```

Comment:
You should be able to extract a lot of these properties to styles (it looks like the `android:text` is one of the only unique attributes on most of these buttons?